app-frontend/src/app/pages/projects/edit/export/export.controller.js

```diff
@@ -0,0 +1,196 @@
+const availableResolutions = [
+    {
+        label: '300m',
+        value: 9
+    },
+    {
+        label: '75m',
+        value: 11
+    },
+    {
+        label: '20m',
+        value: 13
+    },
+    {
+        label: '5m',
+        value: 15
+    },
+    {
+        label: '1m',
+        value: 17
+    }
```

Comment:
hm, might be useful to have the control allow the user to select any zoom level, and give the approx resolution in meters next to the dropdown. The map controls will show the zoom level on the map, so they'll have that for reference